src/commands/GetListCommand.ts

```diff
@@ -26,17 +26,20 @@ export class GetListCommand extends Command {
 			case undefined:
 			case 'bash':
 			case 'fish':
+			case 'zsh':
 				break;
 			default:
-				this.context.stderr.write('Please specify a known shell: bash, fish\n');
+				this.context.stderr.write('Please specify a known shell: bash, fish, zsh\n');
 				return 1;
 		}
 
 		try {
 			const projectNames = [];
 			for (const root of roots) {
 				const dirContents = await fs.readdir(root.path, { withFileTypes: true });
-				const dirs = dirContents.filter(entry => entry.isDirectory());
+				const dirs = dirContents.filter(entry =>
+					entry.isDirectory() && showAllDirectories ? true : !entry.name.startsWith('.')
+				);
```

Comment:
```suggestion
				const dirs = dirContents.filter(
					entry => entry.isDirectory() && (showAllDirectories || !entry.name.startsWith('.'))
				);
```
Just a personal preference really, but boolean literals inside of ternaries feel kinda weird to me.

Reply:
I like this better too, that's what I get for coding past bedtime 😉